at_command.h

```diff
@@ -161,8 +161,8 @@ EXPORT_DECL int at_enqueue_reset(struct cpvt *cpvt);
 EXPORT_DECL int at_enqueue_dial(struct cpvt *cpvt, const char *number, int clir);
 EXPORT_DECL int at_enqueue_answer(struct cpvt *cpvt);
 EXPORT_DECL int at_enqueue_user_cmd(struct cpvt *cpvt, const char *input);
-EXPORT_DECL void at_retrieve_next_sms(struct cpvt *cpvt);
-EXPORT_DECL int at_enqueue_retrieve_sms(struct cpvt *cpvt, int index);
+EXPORT_DECL void at_retrieve_next_sms(struct cpvt *cpvt, int suppress_error);
+EXPORT_DECL int at_enqueue_retrieve_sms(struct cpvt *cpvt, int index, int suppress_error);
```

Comment:
If you want these named, then use an enum. That way the compiler can do some actual checks, instead of depending on the preprocessor.